openapi/openapi.yaml

```diff
@@ -445,6 +638,71 @@ components:
             $ref: '#/components/schemas/Namespace'
 
   responses:
+    Collections:
+      description: 'Response containing a page of Collections'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionsPage'
+
+    Collection:
+      description: 'Response containing a Collection'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Collection'
+
+    CollectionVersion:
+      description: 'Response contain a CollectionVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionVersion'
+
+    CollectionVersions:
```

Comment:
Naming: Considering there are both single and plural forms of schema names, they look too similar.
Also schema is sort of a class, we don't usually use plurals for class or structure names.
Maybe `CollectionVersion` and `CollectionVersionList` is better.

Reply:
My opinion on singular/plural vs singular/singularList kind of changes every day, so I'm flexible.

(operator id) ListCollectionVersions returning a CollectionVersionList (or CollectionVersionsList?) has a symmetry to it.

If we want to change it, should I change it in this pr (for Collection* stuff) now, or get everything in and change plurals->List in bulk in later pr?

Reply:
updated